test/adapters/kagome/cmake/HunterInit.cmake

```diff
@@ -0,0 +1,31 @@
+# specify GITHUB_HUNTER_TOKEN and GITHUB_HUNTER_USERNAME to automatically upload binary cache to github.com/soramitsu/hunter-binary-cache
+# https://docs.hunter.sh/en/latest/user-guides/hunter-user/github-cache-server.html
+string(COMPARE EQUAL "$ENV{GITHUB_HUNTER_TOKEN}" "" password_is_empty)
+string(COMPARE EQUAL "$ENV{GITHUB_HUNTER_USERNAME}" "" username_is_empty)
+
+# binary cache can be uploaded to soramitsu/hunter-binary-cache so others will not build same dependencies twice
+if (NOT password_is_empty AND NOT username_is_empty)
+    option(HUNTER_RUN_UPLOAD "Upload cache binaries" YES)
+    message("Binary cache uploading is ENABLED.")
+else ()
+    option(HUNTER_RUN_UPLOAD "Upload cache binaries" NO)
+    message("Binary cache uploading is DISABLED.")
+endif ()
+
+set(
+    HUNTER_PASSWORDS_PATH "${CMAKE_CURRENT_LIST_DIR}/HunterPasswords.cmake"
+    CACHE FILEPATH "Hunter passwords"
+)
+
```

Comment:
Anything above those lines is not really relevant for use and based on confusing variable names only used on your CI. This does not belong upstream and can all be easily set over some environmental variables.